### PR TITLE
kselftest: use CKI_SELFTEST_URL

### PR DESCRIPTION
**Pull request by liuhangbin**
We should use env CKI_SELFTEST_URL as the download URL.
If not found, the test should skip instead of warn.

Signed-off-by: Hangbin Liu <haliu@redhat.com>

### CODE REVIEW

File: kselftests/runtest.sh [hunk 1]
```diff
@@ -114,8 +114,7 @@ run_test()
 # For upstream kselftest testing, we need a pre-build selftest tar ball url
 install_kselftests()
 {
-	[ ! "$selftests_url" ] && log "no selftests_url" && return 1
-	wget --no-check-certificate $selftests_url -O kselftest.tar.gz
+	wget --no-check-certificate $CKI_SELFTEST_URL -O kselftest.tar.gz
```
Comment (emanchado) [inline kselftests/runtest.sh]: Not really related to this change specifically (I see it was there from before), but why do we need `--no-check-certificate`? Can't we create a proper certificate with Let's Encrypt or something similar?

File: kselftests/include.sh [hunk 2]
```diff
@@ -81,6 +81,18 @@ test_warn()
...
+	echo -e "\n:: [  SKIP  ] :: Test '"$1"'" | tee -a $OUTPUTFILE
+	if [ $JOBID ]; then
+		rstrnt-report-result "${TEST}/$1" SKIP 0
+	else
+		echo -e "\n:::::::::::::::::"
+		echo -e ":: [  ${YEL}SKIP${RES}  ] :: Test '"${TEST}/$1"'"
+		echo -e ":::::::::::::::::\n"
+	fi
+}
+
```
[the remark this answers is not in the document]
Comment (veruu) [reply kselftests/include.sh]: @emanchado I think Beaker interprets any non-zero exit codes as fails and then the test won't be considered skipped (I don't know if we have a standard function that already does this)

[the remark this answers is not in the document]
Comment (emanchado) [reply kselftests/include.sh]: Ooh, the one call to this function (added in this same PR) returns 1...

[the remark this answers is not in the document]
Comment (liuhangbin) [reply kselftests/include.sh]: Not sure if there is a standard function. The main reason I implement this function is to make user able to run it without rstrnt environment.

File: kselftests/runtest.sh (hunk 1 above)
Comment (liuhangbin) [reply kselftests/runtest.sh]: I don't know, should we create the certification on the machine setup stage?

**Comment by emanchado** [comment]
I'm ok with it. @liuhangbin?

**Comment by liuhangbin** [comment]
> I'm ok with it. @liuhangbin?

Me too. Thanks for help reviewing the code.

**Pull request merged**